Gordon360/Services/HousingService.cs

```diff
@@ -737,5 +756,22 @@ public ApartmentApplicationViewModel[] GetAllApartmentApplication()
             }
             return apartmentApplicationArray;
         }
+
+        // I appreciate that we are not typing redundant comments,
+        // but can we have something to make it more clear where these services start?
+        public bool ChangeApplicationDateSubmitted(int applicationID)
+        {
+            SqlParameter appIDParam = new SqlParameter("@APPLICATION_ID", applicationID);
+            DateTime now = System.DateTime.Now;
```

Comment:
For each of the ones that we pass in the current date/time, we could also have the stored procedures use built-in SQL commands to get the current date, so that we don't have to pass it as an SqlParameter (This date stuff is already done in some of our stored procedures to compare the DateModified with the current year). 

This suggestion is more of an efficiency or convenience thing, so at most we might want to make a note of it in the stretch goals of the project board, but there is no need to spend time changing those things right now, especially in light of our other, more urgent goals/milestones.

Reply:
Good point, I will make a note.